src/validation.cairo

```diff
@@ -193,8 +196,38 @@ fn fee_and_merkle_root(block: @Block) -> Result<(u256, u256), ByteArray> {
     Result::Ok((total_fee, merkle_root(ref txids)))
 }
 
-fn validate_coinbase(block: @Block, total_fees: u256) -> Result<(), ByteArray> {
-    //TODO implement
+fn validate_coinbase(self: @ChainState, block: @Block, total_fees: u256) -> Result<(), ByteArray> {
```

Comment:
@maciejka, I added 'self: ChainState' as a parameter to get access to the block_height to be able to computing block reward.

Reply:
Just add `block_height` as parameter. It will be easier to write tests.